.github/workflows/release.yaml

```diff
@@ -2,12 +2,21 @@ name: Release
 on:
   workflow_dispatch:
    inputs:
+     operation:
+       description: "The operation to perform."
+       required: true
+       type: choice
+       default: "build_and_push_images"
+       options:
+         - build_and_push_images
+         - make_okd_community_bundle
```

Comment:
can we have a more descriptive name here please? E.g. "create_okd_release_pr"? :thinking: similar below